todo_app/app.py

```diff
@@ -1,15 +1,35 @@
 from flask import Flask
-
+from flask import request
 from todo_app.flask_config import Config
+from flask import render_template
+from todo_app.data import session_items
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+@app.route('/todo')
 def index():
-    return 'Hello World!'
+    items = session_items.get_items()
+    return render_template('Index.html', items=items)
 
+@app.route('/<id>')
+def task(id):
+    item = session_items.get_item(id)
+    return render_template('single_item.html', item=item)
+
+@app.route('/add', methods=['POST', 'GET'])
+def add():
+    if request.method == 'POST':
+        title = (request.form['title'])
+        session_items.add_item(title)
+        return index()
```

Comment:
You should `return redirect(url_for('index'))`
This means the browser gets a redirect response and will actually navigate to localhost:5000/todo
So the URL the user sees is correct and e.g. they can refresh the page without resubmitting the form.